### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by RominaFior**

### CODE REVIEW

File: staff/RominaFior/js/challenges/scripts/toRomanNumeral.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+function toRomanNumeral(num) {
+    
+    if (typeof num !== 'number' )
+        throw Error('input num is not a number');
+    
+    switch (num) {
+        
+    
+    case 1:
+        num = "I";
```
Comment (alexgbor) [inline staff/RominaFior/js/challenges/scripts/toRomanNumeral.js:12]: use `return "I"` instead of `num = "I"`

Comment (Asix94) [reply staff/RominaFior/js/challenges/scripts/toRomanNumeral.js:12]: @alexgbor Why, you need only one return per funtion

File: staff/RominaFior/js/challenges/scripts/toRomanNumeral.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        break;
+    case  7:
+        num = "VII";
+        break;
+    case 8:
+        num = "VIII"
+        break;
+    case 9:
+        num = "IX"        
+    }
```
Comment (Asix94) [inline staff/RominaFior/js/challenges/scripts/toRomanNumeral.js]: CAUTION, you need case default

File: staff/RominaFior/js/challenges/scripts/toRomanNumeral.js (hunk 1 above)
Comment (alexgbor) [reply staff/RominaFior/js/challenges/scripts/toRomanNumeral.js:12]: 🤔 Hm.. true!

**Review by agandia9 — COMMENTED** [review]
ok!

**Pull request merged**